### ISSUE TITLE
[release/2.0] Newly added 1-1 Channels are not visible until relaunch(Newly added group Channels are never visible after relaunch)

### ISSUE THREAD

**Issue by shireeshMs** [GetStream/stream-chat-swift#127]
## What did you do?
I am trying to start new chat with user

## What did you expect to happen?
it should create chat 1-1 and should reflect on chat list with updated chats .

Let's say if I select A from users list and tap on send ,this should check whether this is new request to new user or request to existing user with whom chat is already initiated before .

After above operation , ChannelViewController  should be reflected with changes .

In short ChannelViewController should reflect that new user has been added to ChannelViewController .
## What happened instead?
It created a chat but does not updated the ChannelViewController with new entry of new chat . 
## GetStream Environment
**GetStream Chat version:**release/2.0
**iOS version:**13.3
**Swift version:**5
**Xcode version:**11.3.1
**Device:**iPhone/Simulator

**Comment by buh** [comment]
Hi @shireeshMs. This issue was fixed and now in review PR #126

**Comment by shireeshMs** [comment]
No it is not still updating the new channels I created ,don't where they go .
not able to get user presence

**Comment by shireeshMs** [comment]
first I tried to create a group 
![Simulator Screen Shot - iPhone 11 Pro Max - 2020-03-23 at 15 25 28](https://user-images.githubusercontent.com/37240061/77304611-b450e880-6d1a-11ea-899a-63a5b71d9f60.png)

Second it load as usual 
![Simulator Screen Shot - iPhone 11 Pro Max - 2020-03-23 at 15 25 32](https://user-images.githubusercontent.com/37240061/77304635-bfa41400-6d1a-11ea-8572-44bbb971d2f2.png)

Third no change on my chat list 
![Simulator Screen Shot - iPhone 11 Pro Max - 2020-03-23 at 15 25 38](https://user-images.githubusercontent.com/37240061/77304679-ccc10300-6d1a-11ea-89f0-73a6f4f28903.png)

**Comment by shireeshMs** [comment]
?

**Comment by b-onc** [comment]
Hello @shireeshMs ,
We're investigating this issue and will get back to you as soon as possible.

**Comment by shireeshMs** [comment]
@b-onc any update on this?

**Comment by shireeshMs** [comment]
@b-onc hello , today I was able to see that channel had been created but not able to see on chat list

**Comment by shireeshMs** [comment]
@b-onc pinging you again .

**Comment by ronnieguha** [comment]
@b-onc I am noticing this on web (sometimes) as well. On Android, I don't think (trying to confirm now) I see this issue. Is this swift related or API related?

**Comment by shireeshMs** [comment]
@b-onc hola,
I was looking to ask you something on this issue regarding it solution.

**Comment by ronnieguha** [comment]
@b-onc I think after the 2.0 release, chat updates and internet connectivity issues have really set things back. For now, we will proceed with Android and web.. and hold off on iOS. Any idea when the recent connectivity issues and chat update issues will be released? We can look at releasing an iOS app then.

**Comment by b-onc** [comment]
Hello @ronnieguha ,
I'm very sorry to hear that.
I assume by "chat updates" you mean this issue, which we're still investigating.
The connectivity issues should be fixed with 2.0.1, can you try that version and let me know?

**Comment by ronnieguha** [comment]
@b-onc no worries. I am sure you guys will fix it. this is going to be an AMAZING app - thanks to you guys. We are already using it *every day* on PWA, desktop, web browser, android, iOS haha.

**Comment by b-onc** [comment]
Hello @shireeshMs ,

You should subscribe to events
```swift
[.notificationInvited, .notificationInviteAccepted, .notificationInviteRejected, .memberUpdated]
```
and when you get a notification, depending on the result, you can call
```swift
presenter.reload()
```
inside ChannelsViewController.

You can check our Example project for a sample implementation.

**Comment by shireeshMs** [comment]
Hello Bahadir,
Yes i guess i after taking pull from master branch ,now it is working as
expected .
Thank you 😊

**Comment by b-onc** [comment]
Hello @shireeshMs ,
Glad to hear that! I'm closing this issue since I understood it as resolved.

**Comment by shireeshMs** [comment]
No again after this pod install ,we are back recent chat never get updated
